src/main/java/org/jitsi/jigasi/transcription/VoskTranscriptionService.java

```diff
@@ -64,15 +64,34 @@
 
     private final static String EOF_MESSAGE = "{\"eof\" : 1}";
 
-    private final String websocketUrl;
+    private String websocketUrlConfig;
+
+    private String websocketUrl;
 
     /**
-     * Create a TranscriptionService which will send audio to the google cloud
+     * Assigns the websocketUrl to use to websocketUrl by reading websocketUrlConfig;
+     */
+    private void generateWebsocketUrl(Participant participant)
+        throws org.json.simple.parser.ParseException
+    {
+        if (!supportsLanguageRouting())
+        {
+            websocketUrl = websocketUrlConfig;
+        }
+
+        org.json.simple.parser.JSONParser jsonParser = new org.json.simple.parser.JSONParser();
+        Object obj = jsonParser.parse(websocketUrlConfig);
+        JSONObject languageMap = (JSONObject) obj;
+        websocketUrl = languageMap.optString(participant.getSourceLanguage());
```

Comment:
Does this code supports the old style of config?

`org.jitsi.jigasi.transcription.vosk.websocket_url=ws://localhost:2700`?

Reply:
> Do we default to some language if there is no match?

Hmm no, but I will set `"en"` as default, as most users would have their interface language set to English.

> Does this code support the old style of config?

Yes, it will detect that it's not JSON and use the string as the WebSocket URL directly.